10-walkthrough-5.Rmd

```diff
@@ -8,74 +8,229 @@ knitr::opts_chunk$set(echo = TRUE, message = F, warning = F, cache = F)
+A common situation encountered when using data for analyzing the education sector, particularly by analysts who are not directly working with schools or districts, is the prevalence of publicly available, aggregate data. [Aggregate data refers to numerical or non-numerical information that is (1) collected from multiple sources and/or on multiple measures, variables, or individuals and (2) compiled into data summaries or summary reports, typically for the purposes of public reporting or statistical analysis.](https://www.edglossary.org/aggregate-data/) Example of publicly available aggregate data include school-level graduation rates, state test proficiency scores by grade and subject, or averaged survey responses.
 
-These datasets are large, lagging, and often suppressed to protect privacy. Because of their coarseness, they can be difficult to use in decision-making. However, these datasets are available to all analysts for gleaning insights about education and can be used for landscape analyses or to supplement other analyses.
+Aggregated datasets are essential both for accountability purposes and for providing useful information about schools and districts to those who are monitoring them. For example, district administrators might aggregate row-level (also known as individual-level or student-level) enrollment reports over time. This allows them to see how many students enroll in each school, in the district overall, and any grade-level variation. Depending on their state, the district administrator might submit these aggregate data to their state education agency for reporting purposes. These datasets might be posted on the state's department of education website for anybody to download and use.
+
+Federal and international education datasets provide additional context for evaluating education systems. In the US, some federal datasets aim to consolidate important metrics from all states. This can be quite useful because each state has its own repository of data and to go through each state to download a particular metric is a significant effort. However, the federal datasets often have more stringent data requirements than the states, so the datasets may be less useable. The federal government also funds assessments and surveys which are disseminated to the public.
+
+For education data practicioners, these reports and datasets can be analyzed to answer questions related to their field of interest. However, publicly available, aggregate datasets are large, lagging, and often suppressed to protect privacy. Because of their coarseness, they can be difficult to interpret and use. Generally, aggregated data are generally used to surface of broader trends and patterns in education as opposed to diagnosing underlyiing issues or making causal statements. It is very important that we consider the limitations of aggregate data before analyzing them.
```

Comment:
minor, describe what is meant by lagging here?

Reply:
I will add a definition!